### PR TITLE
Enable launch_stack tests

### PR DESCRIPTION
**Pull request by manishtomar**
Now that https://github.com/rackerlabs/mimic/issues/513 is managed with #1835

### CODE REVIEW

**Comment by manishtomar** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
@dragorosson review please?

File: otter/integration/lib/autoscale.py [hunk 1]
```diff
@@ -388,7 +388,8 @@ def record_results(resp):
             return rcs
 
         def send(image_id):
-            if image_id is not None:
+            lc_type = self.group_config["launchConfiguration"]["type"]
+            if image_id is not None and lc_type == "launch_server":
                 self.group_config = update_in(
```
Comment (dragorosson) [inline otter/integration/lib/autoscale.py]: This has a hacky feel to it, but okay. At least it works!

**Comment by dragorosson** [comment]
LGTM

File: otter/integration/lib/autoscale.py (hunk 1 above)
Comment (manishtomar) [inline otter/integration/lib/autoscale.py]: Yeah. Unfortunately the whole getting image id in trial tests is hacky :(

**Pull request merged**